egs2/librispeech/asr1/conf/transformer_config.yaml

```diff
@@ -0,0 +1,47 @@
+config: conf/transformer_config.yaml
+print_config: false
+log_level: INFO
+max_epoch: 100
+grad_clip: 5.0
+accum_grad: 4
+batch_size: 15
+batch_type: seq
+optim: adam
+optim_conf:
+    lr: 10.0
+scheduler: noamlr
+scheduler_conf:
+    model_size: 512
+    warmup_steps: 50000
+e2e_conf:
+    ctc_weight: 0.3
+    ignore_id: -1
+    lsm_weight: 0.1
+    length_normalized_loss: false
+    report_cer: false
+    report_wer: false
+    sym_space: <space>
+    sym_blank: <blank>
+use_preprocessor: true
+token_type: bpe
+bpemodel: data/token_list/bpe_unigram5000/model.model
+non_linguistic_symbols: null
+normalize: global_mvn
+normalize_conf:
+    stats_file: exp/asr_stats/train/feats_stats.npz
```

Comment:
```suggestion
```

Configuration file is not allowed to include file path and also the options specified by the shell script is not needed.